Nguyen_Ken/Assignments/great_number_game/server.py

```diff
@@ -6,6 +6,41 @@
 
 @app.route('/')
 def index():
+    session['random'] = int(random.randrange(0,101))
+
+    #reset session
+    if 'highDisplay' in session:
+        session.pop('highDisplay')
+    if 'lowDisplay' in session:
+        session.pop('lowDisplay')
+    if 'correctDisplay' in session:
+        session.pop('correctDisplay')
+    if 'formHide' in session:
+        session.pop('formHide')
+    if 'tooHigh' in session:
+        session.pop('tooHigh')
+    if 'tooLow' in session:
+        session.pop('tooLow')
+    if 'correct' in session:
+        session.pop('correct')
+    if 'guess' in session:
+        session.pop('guess')
+
```

Comment:
What do you mean? 

Sent from Yahoo Mail on Android 
 
  On Tue, Sep 13, 2016 at 7:50 AM, authman<notifications@github.com> wrote:   
In Nguyen_Ken/Assignments/great_number_game/server.py:
> +        session.pop('highDisplay')
> +    if 'lowDisplay' in session:
> +        session.pop('lowDisplay')
> +    if 'correctDisplay' in session:
> +        session.pop('correctDisplay')
> +    if 'formHide' in session:
> +        session.pop('formHide')
> +    if 'tooHigh' in session:
> +        session.pop('tooHigh')
> +    if 'tooLow' in session:
> +        session.pop('tooLow')
> +    if 'correct' in session:
> +        session.pop('correct')
> +    if 'guess' in session:
> +        session.pop('guess')
> +

session.clear() sessions are just dictionaries

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.